docs/endpoints/multi_user.rst

```diff
@@ -486,6 +486,15 @@ Every template also has access to the following variables:
   submitting the task request, such as Python version. See |UserRuntime| for a complete
   list of available information.
 
+- ``mapped_identity``: Contains information about the user's mapped identity. The following
+  fields are available:
+
+  - ``mapped_identity.local.uname``: Local user's username
+  - ``mapped_identity.local.uid``: Local user's ID
+  - ``mapped_identity.local.gid``: Local user's primary group ID
+  - ``mapped_identity.local.groups``: Names of all groups the local user is a member of
+  - ``mapped_identity.globus.id``: Matched Globus identity ID
+
```

Comment:
This is precise (good!); suggest adding an example usage of it in template.  I'm recognizing we don't for `parent_config`, but I'm thinking we oughta.  Can lead the way here.

Reply:
Good idea.